tests/cpp/test_matmul_aten_evaluation.cpp

```diff
@@ -393,4 +408,92 @@ TEST_F(MatmulATenEvaluationTest, LinearWithBias) {
 
   EXPECT_TRUE(at::allclose(out[0], out_ref));
 }
+
+TEST_P(ATenNodesParametrizedTest, MatmulNodeConcrete) {
+  auto fusion = std::make_unique<Fusion>();
+  FusionGuard fg(fusion.get());
+
+  std::vector<int64_t> a_shape = std::get<0>(GetParam());
+  std::vector<int64_t> b_shape = std::get<1>(GetParam());
```

Comment:
```suggestion
  const auto& [a_shape, b_shape] = GetParam();
```

C++17 calls this structured binding declaration fyi.